README.md

```diff
@@ -285,12 +286,8 @@ signIn.renderEl(
   },
 
   function error(err) {
-    // The widget will handle most types of errors - for example, if the user
-    // enters an invalid password or there are issues authenticating.
-    //
     // This function is invoked with errors the widget cannot recover from:
-    // 1. Known errors: CONFIG_ERROR, UNSUPPORTED_BROWSER_ERROR, OAUTH_ERROR
-    // 2. Uncaught exceptions
+    // Known errors: CONFIG_ERROR, UNSUPPORTED_BROWSER_ERROR, OAUTH_ERROR, REGISTRATION_FAILED
```

Comment:
shall we briefly mention that other errors will pop up through `afterEvent`?

Reply:
We already describe that [below](https://github.com/okta/okta-signin-widget/pull/594/files#diff-04c6e90faac2675aa89e2176d2eec7d8R1337). I've changed this text to do a better job of calling this out.

Also, I've created a [follow-up task](https://github.com/okta/okta.github.io/issues/2527) in our "guides" to showcase this new behavior.